deepspeed/runtime/zero/stage_1_and_2.py

```diff
@@ -588,6 +588,10 @@ def _configure_moe_settings(self):
         assert self.ep_process_group is not None, "Expert parallel group should be configured with MoE"
 
     def _update_model_bit16_weights(self, group_index):
+        # Work around due to bug in torch_npu, @see https://gitee.com/ascend/pytorch/pulls/6484
+        # Remove me after torch_npu fixed.
```

Comment:
FYI, this bug has been addressed :)

Reply:
> FYI, this bug has been addressed :)

Thank you. This PR will be closed because all modifications are about torch_npu.